en/about/index.md

```diff
@@ -8,10 +8,15 @@ image: "/banner/aboutus.jpg"
+
+The Walking Machine club is, before everything, a way to learn. Its members gain knowledge through practice in the different fields of ingeeneering. The scientific club has also a social mission, to help citizen with reduced mobility, due to the nature of the competition in which it participates. By developping robots able to accomplish simple domestic tasks, we seek to make life easier for our elders. On another level, we also want to inspire youth, by showing up to diverse promotional events in highschool and colleges. 
 
 ## PHILOSOPHY
-The club has a very important philosophy of openness. Indeed, any tool or free software is recommended when technical options. This choice is motivated by the hope of universal sharing of knowledge. Interdisciplinary collaboration is also strongly encouraged. This leads to develop more knowledge and brings a knowledge of related fields which, otherwise, would that inaccessible. The members are not separated by discipline, but instead distributed according to different projects. All ideas are considered.
+
+The clubs philosophy is being open to give back to the community. In fact, we will use free software before using proprietary software to accomplish our goals. This choice is motivated by the hope of sharing universal knowledge. We are also a multidisciplinary team with a divers background. This helps us build the best robot possible using a multitude of technologies. This helps to developp knowledge to the largest amount of people in connex fields, which would otherwise be almost unaccesible. Our team members are not separated by disciplins, but by projects, and everyone's ideas are considered. 
+
+
 
 ## ORIGINS
-The club Walking Machine takes its origins in its participation in the SAE Walking Machine competition. He participated in the competition for more than ten years. The club was founded in 1995 and has, Since its foundation, designed the entire creation and design of an autonomous mobile robot. The club is the oldest Club of Robotics still function to this day at ETS. Through the years and the changes in competition, the club has evolved, but still retains the same innovative spirit.
+
+The Walking Machine club used to participate in the SAE Walking Machine Student Collegiate Design Series. The club was founded in 1995. The original goal of the club was to build an autonomous robot. We are one of the oldest robotic club at ÉTS. Through out the years and different competitions, the club evolved greatly, but its innovative spirit remains.
```

Comment:
Meme commentaire qu'avant:

**Walking Machine** used to... The club was founded in **1995, with the goal to build an autonomous robot.**